v2/product/product-type.rst

```diff
@@ -1,4 +1,68 @@
 Create a product type
 =====================
+For this example we will create a product type for T-shirts
+## First create Product attribute
+First we will need to create a product attribute. We will create two product attributes - 
 
-@todo
+* Size
+* Color
+
+Goto ``admin/commerce/product-attributes``
+
+Click **Add product attribute**
+![](../images/product-attribute-1.png) 
+
+This will open a form.
+![](../images/product-attribute-2.png) 
+
+After saving this form, we are required to fill *attribute values*.
+![](../images/product-attribute-3.png) 
+
+Now save this form.
+
+Do the same for Color attribute.
+
+## Create Product variation
+Now we need a product variation for the image for t-shirts.
+
+Goto ``admin/commerce/config/product-variation-types``
+
+Click **Add product variation type**
+![](../images/product-variation-1.png) 
+
+This will open a form.
+![](../images/product-variation-2.png) 
+
+Now we will add image field to our product variation.
```

Comment:
"Now we will add ***an*** image field..."

Discuss **why** you are choosing to put the image field on the product variation and not the product. In fact, call that out here in a little aside, give a good reason you would want to choose one over the other, depending on circumstances.